### ISSUE TITLE
Why removed reverberation?

### ISSUE THREAD

**Issue by girtabb** [ShanonPearce/ASH-Listening-Set#7]
I'm curious if there's a possibility that larger rooms could also come alongside a version with their original reverberation, like the Small broadcast studio

**Comment by ShanonPearce** [comment]
That shouldn't be a problem. I will include a set of alternative BRIRs without reverberation reductions for the larger rooms.

**Comment by girtabb** [comment]
Thank you! =)

**Comment by NekoAlosama** [comment]
ShanonPearce didn't answer the question though. I'm assuming it's because the reverberation makes the sound bad or unrealistic (e.g. you're listening in a small room, but your BRIR comes from a very large room), but we wouldn't know until ShanonPearce includes it.

**Comment by ShanonPearce** [comment]
The aim of the reverberation reductions was to reduce the amount of distracting late reflections that were present in the larger rooms. As a result there are differences in perceived room size. 

I have now added alternative BRIRs without the reverberation reductions for most of the larger rooms so the different versions of the rooms can be compared.

**Comment by girtabb** [comment]
Hey ShanonPearce! Thanks for uploading these versions. However, I'm a little confused. The versions you added do add more reverberation, but, from what I know, there are supposed to be a lot more reflections I believe (refer to the video, where I compare and use a file from the WDR website's original samples, in which sound only comes from one angle I think). Now I'm just curious if there are more things you consciously do to these samples to create these IR files or if this is unexpectedly happening. Correct me if I understand something incorrectly, I'm not much of an expert yet.

https://github.com/ShanonPearce/ASH-Listening-Set/assets/131411046/32852184-bc51-4495-a6ed-977beda9c985

**Comment by ShanonPearce** [comment]
I had another look into my script and there is actually still some windowing active that occurs in the tail end of the IR, reducing the overall reverberation time. I will leave the new BRIRs as they are though as removing this windowing will cause a larger mismatch in decay time between the higher frequencies and low frequencies (which are integrated from a separate source and have shorter decay time). 

To avoid any further confusion I have updated the naming of the 2 sets of BRIRs to have "Reduced_Reverb" and "More_Reverb"

**Comment by girtabb** [comment]
So if I understand this correctly, you remove some late reflections or apply "windowing" in order to maintain low-frequency extension and relatively similar decay times across the whole frequency response?

**Comment by girtabb** [comment]
Yeah, I noticed you didn't add the Large broadcast studio. I assume that'd be too much of a hassle to try to make it consistent across the whole spectrum, right?

**Comment by ShanonPearce** [comment]
> Yeah, I noticed you didn't add the Large broadcast studio. I assume that'd be too much of a hassle to try to make it consistent across the whole spectrum, right?

Yeah currently it would be difficult to include the Large broadcast studio in this dataset due to its long reverberation times and a lack of data for low frequency extension. I will look into including it in a future release along with versions of the other large rooms having their original reverberation times. I will first need to improve low frequency extension for rooms containing long reverberation times.

**Comment by girtabb** [comment]
Thanks a lot! In the end, there shouldn't be a problem if you just add "experimental" HRIRs that lack things like bass extension, right?
